client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getEditorNewPostPath } from 'state/ui/editor/selectors';
+
+const TodoLinks = steps =>
+	map( steps, ( { label, url }, stepName ) => (
+		<div key={ stepName } className="steps__summary-entry todo">
+			<a href={ url }>{ label }</a>
+		</div>
+	) );
+
+export const TodoLinksConnected = connect( ( state, { siteId, siteSlug, translate } ) => ( {
+	THEME: {
```

Comment:
BTW I just realized our object keys are string literals here, instead of `[ STEPS.THEME ]` like consts 😵

Reply:
I was actually mistaken -- it never was any different, see e.g. https://github.com/Automattic/wp-calypso/pull/21756/files